modules/react/button/lib/parts/ButtonLabelIcon.tsx

```diff
@@ -26,7 +26,6 @@ const iconSizes: Record<ButtonSizes, number> = {
 };
 
 export const ButtonLabelIcon = createComponent('span')({
-  displayName: 'ButtonLabelIconNew',
```

Comment:
Removing this name as we don't really need it, and it's messing up some component tracking.